software/03.Prisma/README.md

```diff
@@ -1,256 +1,112 @@
-# Workshop 3 - GraphQL API with Prisma2
+# Initial Setup: PostgreSQL and Express.js with Prisma
 
-In this workshop, we will learn how to create GraphQL APIs using [Prisma2](https://www.prisma.io/), Nexus-Prisma and Apollo.
+## Step 1: Installing Necessary Tools
 
-## Step 0: initialization
+Make sure Node.js and npm are installed on your machine. Then, install the following dependencies:
 
-All the required information to install the workshop dependencies are available in [SETUP.md](./SETUP.md).
+1. PostgreSQL: Download and install PostgreSQL from the official website.
+2. Create a PostgreSQL database (e.g., `workshop`).
 
-## Step 1: Retrieve posts
+## Step 2: Initializing the Express.js Project
 
-You saw in the setup a query that retrieves all users from the database. Now, do the same thing but to get all the posts.
+1. Create a new folder for your project and open it in the terminal.
+2. Run the following commands:
```

Comment:
You should create a SETUP.md file with the installation steps and explain a little what they do and then add a link to the actual README so they can click and go back.